db/schema_tables.cc

```diff
@@ -1001,8 +1001,7 @@ future<> merge_schema(sharded<db::system_keyspace>& sys_ks, distributed<service:
         co_return;
     }
     co_await with_merge_lock([&] () mutable -> future<> {
-        bool flush_schema = proxy.local().get_db().local().get_config().flush_schema_tables_after_modification();
```

Comment:
Need to mark flush_schema_tables_after_modification as Unused in config.cc and remove ti from the rest of the source tree. Could be done as a dollow-up.

Reply:
https://github.com/scylladb/scylladb/issues/20578